### DISCUSSION TITLE
More details on “rate limit exceeded” error

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#43987] [Copilot News and Announcements]
Hi there,

This week, we've had to take down some of our Copilot resources for maintenance. While we did our best to source extra capacity elsewhere, we've been forced to limit some free users for the period of time where one of our busiest models is out of commission.

We expect that this will cease to impact your use of Copilot after the 16th of January. In the meantime, you will have less frequent suggestions, but you should still be getting suggestions - just with rate limits in between. You may see these errors specifically if you look in the output pane or logs for your IDE.

Thanks for your patience and apologies for the inconvenience caused.

**Comment by soldni** [comment]
@Akash1134 Thank you for the update. Is it possible to let users upgrade to the paid plan? I would gladly sign up for the higher tier to get rate priority, but, since I am "elegible for the free tier" in the Copilot pane in Settings, I don't see an option to do so.

**Reply by AngelOnFira** [reply]
This would be helpful for me too :100:

**Reply by ettaboyle** [reply]
Thank you for the feedback @soldni & @AngelOnFira As of today, there is no upgrade path from free to paid. I've shared this thread with the copilot team and am also adding the "product feedback" label to make sure they see this as well.

**Reply by asportnoy** [reply]
@ettaboyle @Akash1134 There definitely needs to be a way to upgrade to the paid version if the free version is going to have limitations like these, even if temporary. I assumed that the version we're given would be the same as the paid version, and I'm sure most other free copilot users assumed that was the case as well. If that's not the case, I'd much rather pay the $10/mo to not have to deal with things like this.

**Comment by MulleDK19** [comment]
I barely get suggestions as a paid user. The VS 2022 implementation is still FUBAR.

**Reply by tilmann1** [reply]
@MulleDK19 Which version of VS 2022 and the Copilot extension are you on? Thanks!

**Comment by barum** [comment]
I am a paid user but I get the message. Linux, VSCode. When I try to sign in, I get the error mssage and VSCode closes.

**Reply by BillBu** [reply]
Each time I open VSCode (on Mac) I'm getting "[ERROR] [default] [2023-01-16T09:44:21.035Z] GitHub Copilot could not connect to server. Extension activation failed: "Response code 403 (rate limit exceeded)"

**Reply by Akash1134** [reply]
Hi @BillBu Try closing and reopening your IDE, logout of Copilot and then log back in to see how it goes. Thanks!

**Reply by barum** [reply]
I am still having this issue:
Updated VScode. And extension. 

I am on Linux centos. 


Each time I open VSCode (on Mac) I'm getting "[ERROR] [default] [2023-01-16T09:44:21.035Z] GitHub Copilot could not connect to server. Extension activation failed: "Response code 403 (rate limit exceeded)"

**Comment by BillBu** [comment]
Already tried that. Also uninstalled and reinstall the extension.
